packages/design-system-scripts/gulp/docs/extractReactData/index.js

```diff
@@ -21,7 +21,7 @@ async function extractReactProps(sourceDir, options) {
     gulp
       .src([
         `${sourcesGlob}/**/*.{jsx,tsx}`, // React props
-        `!${sourcesGlob}/**/*{.test,.spec,.d}.{js,jsx,ts,tsx}`,
+        `!${sourcesGlob}/**/*{.stories,.test,.spec,.d}.{js,jsx,ts,tsx}`,
```

Comment:
Added this to resolve these, doesn't effect anything else, ok to keep in here?

![image](https://user-images.githubusercontent.com/783820/145600093-e10ac445-504b-406a-bc42-f51f9a1dc6e4.png)

Reply:
We don't need to run .stories files through the docs site prop doc generator

Reply:
Nice! Thank you